settings_dialog.py

```diff
@@ -43,6 +44,8 @@ def __init__(self, *args, **kwargs) -> None:
         self.max_notes_edit = CroProSpinBox(min_val=10, max_val=10_000, step=50, value=config.max_displayed_notes)
         self.hidden_fields = ItemEditBox("Hidden fields", initial_values=config.hidden_fields)
         self.web_timeout_spinbox = CroProSpinBox(min_val=1, max_val=999, step=1, value=config.timeout_seconds)
+        self.sentence_min_length = CroProSpinBox(min_val=0, max_val=196, step=1, value=config.sentence_min_length)  # Currently longest sentence has a length of 196 letters (Shirokuma Cafe Outro full sub)
+        self.sentence_max_length = CroProSpinBox(min_val=0, max_val=197, step=1, value=config.sentence_max_length)
```

Comment:
Do you really need to hardcode 196 and 197 here? These values may change if the remote sentence bank gets updated so it's probably safer to set max_val to a larger number.